packages/tasks/src/tasks/text-to-speech/data.ts

```diff
@@ -4,8 +4,8 @@ const taskData: TaskDataCustom = {
 	canonicalId: "text-to-audio",
 	datasets: [
```

Comment:
I currently don't see any datasets on the task page, even though there's supposed to be LibriTTS, am I missing something ?

Reply:
Maybe due to the rename to https://huggingface.co/datasets/keithito/lj_speech